Twig/SecurityExtension.php

```diff
@@ -3,27 +3,33 @@
 namespace JMS\SecurityExtraBundle\Twig;
 
 use JMS\SecurityExtraBundle\Security\Authorization\Expression\Expression;
+use JMS\SecurityExtraBundle\Exception\InvalidArgumentException;
+use Symfony\Component\Security\Core\Authorization\AuthorizationChecker;
 use Symfony\Component\Security\Core\SecurityContextInterface;
 
 class SecurityExtension extends \Twig_Extension
 {
-    private $context;
+    private $authorizationChecker;
 
-    public function __construct(SecurityContextInterface $context)
+    public function __construct($authorizationChecker)
```

Comment:
I would appreciate a doc comment here too which explains what to expect, i.e. security context or authorization checker. Could you add that?